common/src/main/java/com/wynntils/models/gear/type/GearType.java

```diff
@@ -19,87 +17,116 @@
 import net.minecraft.world.item.component.CustomModelData;
 
 public enum GearType {
-    SPEAR(ClassType.WARRIOR, Items.IRON_HORSE_ARMOR, 437, 497, 0),
-    WAND(ClassType.MAGE, Items.IRON_HORSE_ARMOR, 308, 372, 1),
-    DAGGER(ClassType.ASSASSIN, Items.IRON_HORSE_ARMOR, 244, 307, 2),
-    BOW(ClassType.ARCHER, Items.IRON_HORSE_ARMOR, 182, 243, 3),
-    RELIK(ClassType.SHAMAN, Items.IRON_HORSE_ARMOR, 373, 436, 4),
+    SPEAR(ClassType.WARRIOR, ModelSupplier.forKey("spear_min"), ModelSupplier.forKey("spear_max"), 0),
```

Comment:
I had missed ModelSupplier. I am not really happy with that API. The factory method should be ` Services.CustomModel.getSupplierForKey()` or something like that instead. Now the CustomModel service is just completely hidden from view... I'm not saying you need to fix it in this PR, but I won't hold it against you if you do. :)